.idea/misc.xml

```diff
@@ -1,5 +1,32 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project version="4">
+  <component name="EntryPointsManager">
```

Comment:
yes, the file will change according to your local system.
But don't send these as a file of your PR otherwise when other people will merge changes then that can be an error on there system. :)